src/torchcodec/_core/BetaCudaDeviceInterface.cpp

```diff
@@ -53,74 +53,6 @@ pfnDisplayPictureCallback(void* pUserData, CUVIDPARSERDISPINFO* dispInfo) {
 }
 
 static UniqueCUvideodecoder createDecoder(CUVIDEOFORMAT* videoFormat) {
```

Comment:
All checks below have been moved to the new `bool nativeNVDECSupport(const SharedAVCodecContext& codecContext);`